target/linux/mvebu/config-4.9

```diff
@@ -60,6 +60,8 @@ CONFIG_BLK_DEV_LOOP=y
 CONFIG_BLK_DEV_SD=y
 CONFIG_BLK_MQ_PCI=y
 CONFIG_BOUNCE=y
+CONFIG_BTRFS_FS=y
```

Comment:
Quick comparison
ext4 + F2FS + btrfs: 2.426 MiB
ext4 + F2FS (current image): 2.103 MiB
(without ClearFog support): 1.927 MiB

I had hoped to maintain a simple flashing process, however seeing this I will look at the possibility of using a ClearFog-like eMMC layout and performing a somewhat more manual initial flash via built-in OpenWrt-based Omnia recovery partition.